City-of-Helsinki/helsinki-paatokset
# PP-0: Package.lock and dependabot updates

This should fix a bunch of dependabot warnings. Also updates node-sass to version 7, so SCSS compiling should be tested especially in the subtheme.

**To test**
- Checkout branch
- Go to the theme folder `public/themes/custom/helfi_paatokset`
- Run `nvm use` (if you don't have the correct node version, use nvm to install it)
- Run `npm install`
  - This _should_ run without errors. If it doesn't, try deleting `package-lock.json` and `node_modules`
  - Comment on this PR about what changed in your package-lock.json if you had to do this
- Run `npm run build`, which should complete without errors, and should not cause the CSS file to be any different
- Make sure the `npm run dev`, `npm run lint` and `npm run lint:js` commands work as intended
- Make some changes to SCSS files and run `npm run dev` and `npm run build`
- Also create some new SCSS files and make sure they are picked up too by the globber
